lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -1714,15 +1714,19 @@ def init_api_client
     def api_client
       if @use_grpc
         ssl_creds = GRPC::Core::ChannelCredentials.new
-        authentication = Google::Auth.get_application_default
-        creds = GRPC::Core::CallCredentials.new(authentication.updater_proc)
+        authentication = Google::Auth.get_application_default(LOGGING_SCOPE)
```

Comment:
I'll let @qingling128 comment on whether we need an equivalent change to the tests ([`GoogleCloudOutputGRPCTest::GoogleCloudOutputWithGRPCMock.api_client`](https://github.com/GoogleCloudPlatform/fluent-plugin-google-cloud/blob/master/test/plugin/test_out_google_cloud_grpc.rb#L246)).

Reply:
Since we have added the scope in the plugin, we should change it accordingly in `GoogleCloudOutputGRPCTest::GoogleCloudOutputWithGRPCMock.api_client` as well. 

The way credentials are constructed in the mock should be identical to:
```
        ssl_creds = GRPC::Core::ChannelCredentials.new
        authentication = Google::Auth.get_application_default(LOGGING_SCOPE)
        creds = GRPC::Core::CallCredentials.new(lambda do |a_hash|
          authentication.apply(a_hash, use_configured_scope: true)
        end)
        creds = ssl_creds.compose(creds)
```